CONTRIBUTING.md

```diff
@@ -261,8 +261,8 @@ The Nix code which builds our packages also needs some information relating to
 the index-state. This information needs to be new enough to include the
 index-state specified in `cabal.project`. The information is represented by Nix
 flake inputs. You can update these by running:
-- `nix flake update hackageNix` for Hackage
-- `nix flake update CHaP` for CHaP
+- `nix flake lock --update-input hackageNix` for Hackage
+- `nix flake lock --update-input CHaP` for CHaP
```

Comment:
Why is this? `nix flake update hackageNix` works perfectly for me.

Reply:
Sorry, it seems like I have a somewhat ancient version of Nix on my laptop, but I don't want to update it while at the Peras workshop. I will try from my workstation on Monday and revert that commit.